### ISSUE TITLE
multiprocessing support

### ISSUE THREAD

**Issue by jf---** [tpaviot/pythonocc-generator#9]
multiprocessing.Pool would helps to speed up generating the SWIG files linearly

**Comment by jf---** [comment]
speed up is pretty much linear

**Comment by tpaviot** [comment]
The previous generator, based on gccxml, was very long to achieve the generation of swig files (more than one hour AFAIR). Multiprocessing was really useful to speed up generation. Since the move to CppHeaderParser, it's much quicker, around 5mn on a single core usual machine. I did not port the multiprocessing feature because it makes the debugging task a bit more difficult since log messages are mixed together. False by default is a must, I'm not sure I'll ever switch it to True.

**Comment by jf---** [comment]
I'm not advocating for it to run in parallel by default, its makes tracing much more difficult and renders the stdout / stderr output pretty useless. Nice to have though
